scripts/ci-select-xcode.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# For available Xcode versions see:
+# - https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode
+# - https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md
+# - https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md
+
+set -euo pipefail
+
+# 14.3 is the default
+XCODE_VERSION="${1:-14.3}"
```

Comment:
`h`: We must use Xcode 15.x, because in a couple of weeks all apps uploaded to TestFlight must be build with Xcode 15. You could also use https://github.com/XcodesOrg/xcodes to replace this script, as this is now installed per default for the GH action ima `macos-13`, see https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#tools.